types/lhr.d.ts

```diff
@@ -41,6 +41,8 @@ declare global {
       fetchTime: string;
       /** The version of Lighthouse with which these results were generated. */
       lighthouseVersion: string;
+      /** The version of libraries with which these results were generated. Ex: axe-core. */
+      credits: Record<string, string>,
```

Comment:
a new toplevel property for this still makes me cringe a little, do we really foresee ever surfacing anything else here?

I'm still in camp `environment.axeVersion` *ducks*

Reply:
even after my suggestion of `environment.credits` ?

Reply:
`.environment.credits`  preferred to `.credits` for sure :)

I'm just hesitant on credits without any clue what would go into here that doesn't encourage us to start listing every dependency. is there any particular future lib you had in mind we might one day want to put into here?

Reply:
I was thinking anything that informs the results of the audits (so not, for example, lodash):

`js-library-detector`
`third-party-web`
`robots-parser`

(not exhaustive)

Not suggesting we show all in runtime settings. Would need different UI I think for more than one version. axe is the largest so am ok just showing that in runtime setting for now.

Reply:
> third-party-web

I see what you did there ;) sold 🤣 😆